universal-application-tool-0.0.1/app/assets/javascripts/azure_upload.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * This is responsible for uploading a file to Azure blob storage.
+ */
+class AzureUploadController {
+  static UPLOAD_CONTAINER_ID = 'azure-upload-form-component';
+
+  constructor() {
+    const uploadContainer = document.getElementById(AzureUploadController.UPLOAD_CONTAINER_ID);
+    uploadContainer.addEventListener("submit", (event) => this.attemptUpload(event, uploadContainer));
+  }
+
+  getValueFromInputLabel(label: string): string {
+    return (<HTMLInputElement>document.getElementsByName(label)[0]).value;
+  }
+
+  attemptUpload(event: Event, uploadContainer: HTMLElement | null) {
+    event.preventDefault();
+    const azblob = window["azblob"];
+    if (uploadContainer != null) {

```

Comment:
suggestion (blocking): invert the conditional and log an error for the null argument